webClient/src/app/shared/dialog/delete-file/delete-file.component.scss

```diff
@@ -8,6 +8,14 @@
   
   Copyright Contributors to the Zowe Project.
 */
+

```

Comment:
There's a better way to do this while also removing the need for code duplication in tons of files. Try setting a global style that any HTML template can use (maybe in zlux-editor\webClient\src\styles.scss ?)

Additionally, "background: white;" doesn't actually do anything right? Since the modals are already white. But, if they were to chance colors for some reason, maybe after Editor gains a themeing feature or something in the future, then the "X" button would look a little wonky with a white box around it. Try "background: transparent;" ?